testpilot/frontend/static-src/app/views/feedback-view.js

```diff
@@ -1,84 +1,45 @@
-import app from 'ampersand-app';
 import BaseView from './base-view';
```

Comment:
This is being used for the Disable dialog. I suppose I could rename it if it is only used there now.